### PR TITLE
First draft of question and answer

### PR DESCRIPTION
**Pull request by borisrev**
#### Description of Change:
Updating the Intelligence faq to include a commonly asked question.

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [x] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Currents<br>Internal Tools<br>Product Partnerships<br>SMS<br>Customer Lifecycle, Identity and Permissions</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Intelligence<br>User Targeting<br>IAM<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging and Automation<br>Email (Composition and Infrastructure)</td>
  </tr>
</table>
</details>

### CODE REVIEW

**Review by mmwillet — APPROVED** [review]

**Comment by bre-fitzgerald** [comment]
@borisrev after our Slack chat, I've moved this to the "Things to know" section of creating a multivariate and A/B test, and reframed it more as a statement rather than a question. If this continues to grow we can definitely split it into its own FAQ page. 

I also made some edits for clarity/style, can you and @mmwillet take a look to make sure it's still functionally correct? 🙏

**Review by mmwillet — APPROVED** [review]
I read over this and didn't see any critical issues.

File: _docs/_user_guide/engagement_tools/testing/multivariant_testing/create_multivariate_campaign.md [hunk 1]
```diff
@@ -154,6 +154,24 @@ Additionally, when choosing which segment to run your test on, be sure to consid
 As a guide, you likely need around 15,000 users per variant (including the control) to achieve 95% confidence in your test results. However, the exact number of users you need could be higher or lower than that depending on your particular case. For more exact guidance on variant sample sizes, consider referring to [Optimizely's Sample Size Calculator](https://www.optimizely.com/resources/sample-size-calculator/).
 {% endalert %}
 
+### Control and test group assigments and bias
+
+A common question with control and test group assignments is wondering if they can introduce bias to your testing.
+
+The answer is no, due to the way users are assigned to variants or their respective control groups, there is no practical way that user behaviors prior to them receiving a particular campaign or Canvas could vary systematically.
+
+Users are assigned to message variants, Canvas variants, or their respective control groups by concatenating their user ID with the campaign or Canvas ID, taking the modulus of that value with 100, and then ordering users into slices that correspond to the percentage assignments chosen in the dashboard.
```
[the remark this answers is not in the document]
Comment (bre-fitzgerald) [reply _docs/_user_guide/engagement_tools/testing/multivariant_testing/create_multivariate_campaign.md:163]: Hmm okay. My concern with the way it was originally presented was that I couldn't really understand what the answer was until I reread it a few times. Obviously I'm not the target audience, but my intention here was to summarize the answer and then go into more detail. I can go back to your suggestion :)

Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/testing/multivariant_testing/create_multivariate_campaign.md:163]:
```suggestion
A common question with control and test group assignments is wondering if they can introduce bias to your testing.

Users are assigned to message variants, Canvas variants, or their respective control groups by concatenating their (randomly generated) user ID with the (randomly generated) campaign or Canvas ID, taking the modulus of that value with 100, and then ordering users into slices that correspond to the percentage assignments for variants and optional control chosen in the dashboard. So, there is no practical way that users' behaviors prior to the creation of a particular campaign or Canvas could vary systematically between variants and control.

```

File: _docs/_user_guide/engagement_tools/testing/multivariant_testing/create_multivariate_campaign.md [hunk 2]
```diff
@@ -154,6 +154,23 @@ Additionally, when choosing which segment to run your test on, be sure to consid
 As a guide, you likely need around 15,000 users per variant (including the control) to achieve 95% confidence in your test results. However, the exact number of users you need could be higher or lower than that depending on your particular case. For more exact guidance on variant sample sizes, consider referring to [Optimizely's Sample Size Calculator](https://www.optimizely.com/resources/sample-size-calculator/).
 {% endalert %}
 
+### Control and test group assigments and bias
+
+A common question with control and test group assignments is wondering if they can introduce bias to your testing.
+
+Users are assigned to message variants, Canvas variants, or their respective control groups by concatenating their (randomly generated) user ID with the (randomly generated) campaign or Canvas ID, taking the modulus of that value with 100, and then ordering users into slices that correspond to the percentage assignments for variants and optional control chosen in the dashboard. So, there is no practical way that users' behaviors prior to the creation of a particular campaign or Canvas could vary systematically between variants and control.
+
+
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/engagement_tools/testing/multivariant_testing/create_multivariate_campaign.md:163]:
```suggestion

```

**Pull request merged**